providers/github.go

```diff
@@ -73,7 +73,7 @@ func (p *GitHubProvider) hasOrg(accessToken string) (bool, error) {
 	pn := 1
 	for {
 		params := url.Values{
-			"limit": {"200"},
+			"per_page": {"100"},
 			"page":  {strconv.Itoa(pn)},
```

Comment:
You'll need to indent this line a few more characters to fix the linting issue on Travis, the two opening braces should line up
```suggestion
			"page":     {strconv.Itoa(pn)},
```